templates/job_upstart.erb

```diff
@@ -1,4 +1,4 @@
 # FLAG: MANAGED BY PUPPET
 start on <%= @reporting_name %>
 task
-exec setuidgid '<%= @user %>' '/nail/etc/upstart_crons/<%= @title %>'
```

Comment:
'initctl emit --no-wait' will always return zero if the job exists, and is always executed by root.  We want to maintain compatibility by:
1) getting the actual output code of the job itself
2) running the success wrapper as the cron user